server/lib/tuist/billing.ex

```diff
@@ -30,6 +30,17 @@ defmodule Tuist.Billing do
     @unit_prices
   end
 
+  @doc """
+  Gets month-to-date remote cache hits count for an account.
+  Returns an integer, defaulting to 0 when data is missing.
+  """
+  def month_to_date_remote_cache_hits_count(account_id) when is_integer(account_id) do
+    case CommandEvents.month_to_date_remote_cache_hits_count(account_id) do
+      %{remote_cache_hits_count: count} when is_integer(count) -> count
+      _ -> 0
+    end
```

Comment:
similarly should we crash here, too, if we get an unexpected result instead of showing a zero that will be easy to miss?

```suggestion
    %{remote_cache_hits_count: count} = CommandEvents.month_to_date_remote_cache_hits_count(account_id)
```